src/Auth0Client.ts

```diff
@@ -64,7 +64,7 @@ export default class Auth0Client {
       response_mode: 'query',
       state,
       nonce,
-      redirect_uri: this.options.redirect_uri || redirect_uri,
+      redirect_uri: redirect_uri || this.options.redirect_uri,
```

Comment:
why was this flipped? where do `this.options` comes from?

Reply:
@lbalmaceda This is ultimately what was causing the bug. The options passed into `createAuth0Client` (`this.options`) was taking precedence over the options being passed into `loginWithPopup`, `getTokenSilently`, etc. It should be the other way around.